boards/arm/ti_am62x_sk/doc/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _ti_am62x_sk:
+
+AM62x-SK M4F Core
+#################
+
+Overview
+********
+
+The AM62x-SK board configuration is used by Zephyr applications that run on
+the TI AM62x platform. The board configuration provides support for the ARM
+Cortex-M4F CPU and the following devices:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+
+The board configuration also enables support for the semihosting debugging console.
+
+See the `TI AM62X Product Page`_ for details.
+
+Hardware
+********
+
+- Low-power ARM Cortex-M4F
+- Memory
+
+   - 256KB of SRAM
+
+- Debug
+
+   - XDS110 based JTAG
+
+Supported Features
+==================
+
+The ti_am62x_sk configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/ti_am62x_sk/ti_am62x_sk_defconfig
+
+Flashing
+********
+
+`UniFlash`_ command line utility is used to program the flash memory. Only
+elf loading is currently supported.
+
```

Comment:
I am trying to allow the board to be flashed without any vendor specific tools, and so have removed this. Will keep you updated!

Reply:
Sorry my comment was misleading, and not specifically targeted at these lines of code.
A vendor specific command line utility like "UniFlash" is fine.
My preference is only to avoid requiring vendor IDEs, which you had already done.
All I suggest is to provide clear steps that will allow others to easily program the board